kaprekar.js

```diff
@@ -0,0 +1,67 @@
+function hasSimilardigits(digits) {
+    let count = 0;
+    for (let i = 0; i < digits.length - 1; i++) {
+      if (digits[i] === digits[i + 1]) {
+        count++;
+      }
+    }
+    if (digits.length - count == 1) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+  
+  function arrangeInAscendingOrder(digits){
+    return digits
+    .sort((a, b) => {
+      return a - b;
+    })
+    .join("");
+  }
+  
+  function arrangeInDescendingOrder(digits){
+    return digits
+    .sort((a, b) => {
+      return b - a;
+    })
+    .join("");
+  }
+  
+  function applyKaprekar(digits) {
+    let nextNumber = NaN;
```

Comment:
you can directly store `largestNumber - smallestNumber` inside const nextNumber instead of using let and assigning NaN to it before